content/components/local_time.mdx

```diff
@@ -0,0 +1,4 @@
+---
+title: Local time
+componentId: local_time
+---
```

Comment:
Nitpick: Can we use kebab case for the file names?

I think `/components/local-time` is slightly better URL than `/components/local_time`?

Reply:
Hey @colebemis, this PR was meant to be a draft to test out the `componentId` and extract descriptions from the `component-metadata` library. Closing it asap 👍